### ISSUE TITLE
[FEAT] Complete Redesign of Client's UI

### ISSUE THREAD

**Issue by Vimall03** [Vimall03/Alimento#43]
## Description:
This is a level 3 issue, requiring a complete redesign of the client’s end of the website. Please do not request to be assigned to this issue unless you are confident and ready to take on a full redesign.

You have the freedom to use any tools or frameworks such as **Bootstrap** or **Tailwind CSS**, but please ensure:
- All links and buttons work correctly
- No broken links or pages exist after the redesign.

## Initial Step:
Before beginning development, the first requirement is to submit a **Figma design** showcasing the new UI. Based on the design, I will review and assign the issue accordingly.

## Screenshots:
Please provide **before** and **after** screenshots of the pages you've redesigned to visually demonstrate the improvements.

![image](https://github.com/user-attachments/assets/619c29b8-c957-4726-b6e5-dd4248b88bd4)

![image](https://github.com/user-attachments/assets/87de31c2-8840-41fb-99ab-002a6922664d)

![image](https://github.com/user-attachments/assets/c8897b68-874b-4daf-a116-b3a575290f5b)

**Comment by SumitSahu44** [comment]
@Vimall03 can you explain more about it im ready to do this

**Comment by Sourabh782** [comment]
@Vimall03 can u assign it to me I can redesign it

**Comment by Vimall03** [comment]
@SumitSahu44 Sorry, I did not see your comment on this issue. Yes, I'm expecting a complete redesign of the UI here. 
I don't have any image or design on my mind, The only requirement is it should be visually appealing. Something like Zomato.
You have the freedom to design and implement it as long as the backend works as expected

If you are still interested then please post a comment, i ll assign this to you.

**Comment by Vimall03** [comment]
Hey @Sourabh782, If you are interested in the redesign then you can look into issue #44. Its fairly similar to this issue. Please post a comment in issue #44 if you are interested.

**Comment by SumitSahu44** [comment]
@Vimall03 yes i'm intrested page you tell me page title means what data it's contain like home page or something... or tell me what's your website does?

**Comment by Vimall03** [comment]
I have clearly explained the issue in the description. You can look into readme.md for all the information you need. I don't understand what you are asking.

**Comment by SumitSahu44** [comment]
@Vimall03 assign this to me now i'm ready to work on it

**Comment by Vimall03** [comment]
@SumitSahu44 You can get started now, You have the creative freedom to make the design visually appealing, but  **please ensure all current features—such as adding items to cart, search functionality and others—continue to work as expected after the new design is implemented.**

**Comment by SumitSahu44** [comment]
ok sure

**Comment by SumitSahu44** [comment]
hey @Vimall03 is this design is perfect for menu sections 
**Before**
![image](https://github.com/user-attachments/assets/804626e6-915f-46e6-a50e-ea867b319d8d)


**After**
![image](https://github.com/user-attachments/assets/553c16b0-e241-4ac7-aa61-11d7295f2add)

**Comment by Vimall03** [comment]
@SumitSahu44 Looks Amazing.

**Comment by SumitSahu44** [comment]
so any any changes final it?

**Comment by Vimall03** [comment]
Yes, Also make sure the cart is visible.

**Comment by SumitSahu44** [comment]
on same page i think we can add cart icons on nav bar user can click card icon and see all the items

**Comment by Vimall03** [comment]
Okay, will you be able to do that functional ?

**Comment by SumitSahu44** [comment]
yes, i can do it

**Comment by SumitSahu44** [comment]
Yes i can fix it and i saw many frontend issues
![image](https://github.com/user-attachments/assets/6c761874-b5c9-431f-9b49-4dead87e3c58)
![image](https://github.com/user-attachments/assets/b75bbdff-048b-4b24-8f95-eebf488c1e71)

**Comment by Vimall03** [comment]
Yes, The inappropriate rendering in the 2nd image will be fixed in an issue currently assigned to a contributor.

The navbar must be redesigned as well. But let's focus on completing this issue. Once this issue is completed we can focus on more fixes

**Comment by Vimall03** [comment]
Hi @SumitSahu44, what is the progress ?

**Comment by SumitSahu44** [comment]
sorry @Vimall03 for the late response but i will  update you tomorrow with all work done

**Comment by SumitSahu44** [comment]
![image](https://github.com/user-attachments/assets/472575ba-0606-46ab-bc43-08fb8740bb60)
is this good design?

**Comment by Vimall03** [comment]
Can you do something like this ? 
![image](https://github.com/user-attachments/assets/39220de2-033f-4b82-88cf-d948050f85f9)

**Comment by SumitSahu44** [comment]
yes i can but it take too much effort's in same repo it is very difficult

**Comment by Vimall03** [comment]
I understand, That's why I have given this level 3. Also you won't have to design the Track orders page since that will be covered in #96. 
You don't have to exactly add all the details in the reference image, but change the look of the form and the form fields. You can use bootstrap via CDN to quickly finish the work.

**Comment by SumitSahu44** [comment]
ok i will , in these repo i will make 2 pages UI one is done which is (menu of particular restaurant) and another is these shipping address page I'm right ?

**Comment by Vimall03** [comment]
Yes these are the 2 UI changes in this issue.

**Comment by SumitSahu44** [comment]
ok done i will do it today and pull these repo 😉

**Comment by SumitSahu44** [comment]
Hey @Vimall03  Sorry it took me too much time but trust me it will be totally worth it

**Comment by Vimall03** [comment]
@SumitSahu44 Okay, When can i expect the PR ?

**Comment by SumitSahu44** [comment]
@Vimall03 may be today

**Comment by SumitSahu44** [comment]
hey @Vimall03 done please check it and suggest any improvement if needed and add all labels which help me
